hdp/process/load.py

```diff
@@ -1,4 +1,7 @@
 import pandas as pd
+from bs4 import BeautifulSoup
+from bs4 import NavigableString, Tag
+
```

Comment:
Import proper typing suggestions
```suggestion
from typing import List, Tuple, Dict
```